lighthouse-core/report/html/report-styles.css

```
@@ -852,6 +852,19 @@
   vertical-align: middle;
 }
 
+.lh-audit-group--budgets .lh-table tbody tr td:nth-child(4),
```

could we get a comment explainin what `nth-child(4)` and 5 are in case they change in the future?

```suggestion
/* Style the "over budget" columns red. */
.lh-audit-group--budgets .lh-table tbody tr td:nth-child(4),
```